devito/ir/ietxdsl/cluster_to_ssa.py

```diff
@@ -55,9 +55,12 @@ class ExtractDevitoStencilConversion:
 
     eqs: list[LoweredEq]
     block: Block
-    loaded_values: dict[tuple[int, ...], SSAValue]
+    temps: dict[tuple[DiscreteFunction, int], SSAValue]
     time_offs: int
 
+    def __init__(self):
+        self.temps = dict()
```

Comment:
maybe overkill

Reply:
WDYM? It crashes if I don't initialize it; do you mean you prefer another way of initializing and/or passing the dictionary around?

Reply:
ah ok now I see what you mean. Cool